### DISCUSSION TITLE
Issuing GitHub Student Application

### DISCUSSION THREAD

**Discussion by malinga93142** [community/community#57745] [GitHub Education]
Regarding My Application,when will my application get accepted?

**Comment by samus-aran** [comment]
Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
